src/views/Support.vue

```diff
@@ -219,5 +225,90 @@ export default {
 			}
 		},
 	},
+	computed: {
+		getInstalledComponents() {
+			let componentText = "";
+			const devices = this.getWildcardTopics(
+				"openWB/system/device/+/config",
+			);
+			for (const [, deviceConfig] of Object.entries(devices)) {
```

Comment:
`Object.entries()` gibt `key`-`value` Paare zurück. Wenn der `key` hier nicht benötigt wird, sollte besser `Object.values()` verwendet werden.
```suggestion
			for (const deviceConfig of Object.values(devices)) {
```